src/System.Data.Odbc/tests/Helpers.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Data.Odbc.Tests
+{
+    public static class Helpers
+    {
+        public const string OdbcIsAvailable = nameof(Helpers) + "." + nameof(CheckOdbcIsAvailable);
+        public const string OdbcNotAvailable = nameof(Helpers) + "." + nameof(CheckOdbcNotAvailable);
+
+        public static bool CheckOdbcNotAvailable()
```

Comment:
These really should be properties ("OdbcIsAvailable"..), with a static cached if necessary. Methods to get a boolean are more Java-y.